recipe/meta.yaml

```diff
@@ -64,20 +58,70 @@ requirements:
   run_constrained:
     - __osx >={{ MACOSX_DEPLOYMENT_TARGET|default("10.9") }}  # [osx]
 
-test:
-  commands:
-    - test -f ${PREFIX}/lib/libglib-2.0{{ SHLIB_EXT }}  # [not win]
-    - test ! -f ${PREFIX}/lib/libgobject-2.0.la  # [not win]
-    # Check that binaries can run -- instigated by Meson Linux rpath issue
-    - gapplication help  # [linux]
-    - gdbus help
-    - gio version
-    - gio-querymodules .
-    - glib-compile-resources --help
-    - glib-compile-schemas --help
-    - gobject-query --help
-    - gresource help
-    - gtester --help  # [not win]
+outputs:
+  - name: libglib
+    script: install.sh  # [unix]
+    script: install.bat  # [not unix]
+    build:
+      run_exports:
+        - {{ pin_subpackage("libglib") }}
+    requirements:
+      build:
+        - {{ posix }}patch  # [win]
+        - meson
+        - ninja
+        - {{ native }}pkg-config
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        - ninja
+        - python >=2.7
+      host:
+        - libffi
+        - gettext
+        - zlib
+        - pcre
+        - libiconv
+      run_constrained:
+        # Avoid colliding with older glib builds that didn't have a libglib output
```

Comment:
How does this work with older `glib` builds but with the same version number?

Reply:
Actually, it doesn't work. For now I would switch back to `"exact=true"` and only use the `max_pin` once we have a new release where we only have built the split package.

Reply:
`exact=True` wouldn't work because at the time `libglib` is rendered by conda-build it doesn't know the hash of `glib` because it hasn't rerendered `glib` yet

Reply:
Damn. Any good suggestions for alternatives? The only thing I can currently think off is repodata patching but I would only like to use that as a last resort.

Reply:
something like `libglib {{ version }} *_{{ build_number }}`